syz-ci/manager.go

```diff
@@ -879,9 +879,15 @@ func (mgr *Manager) uploadCoverStat(fuzzingMinutes int) error {
 
 	resp, err := mgr.httpGET("/cover?jsonl=1")
 	if err != nil {
-		return fmt.Errorf("failed to get /cover?json=1 report: %w", err)
+		return fmt.Errorf("failed to httpGet /cover?jsonl=1 report: %w", err)
 	}
 	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		sb := new(strings.Builder)
+		io.Copy(sb, resp.Body)
+		return fmt.Errorf("failed to GET /cover?jsonl=1, httpStatus %d: %s",
```

Comment:
The error is ultimately streamed to the dashboard via our HTTP API:

https://github.com/google/syzkaller/blob/e3405ca6695b1c3732c5e380c89f1a741a3fe82a/syz-ci/manager.go#L999-L1003

Do we really want to include the whole downloaded body in there?

Reply:
It's of course good to have many details, but do you really want several MB of details in the textual log messages that we have so far kept small?

Also keep in mind that GAE only accepts 32MB of post data. If the request is larger, it just won't be received.

Maybe cut resp.Body to the first 1KB of data? If there was just an error message, it should be more than enough.

Reply:
Server can set error code only before the body creation. I don't see how in this case we can return 4xx or 5xx with at least part of the valid content.
I expect to see tens-hundreds bytes here.